locale/pt-br/blog/community/transitions.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Transições
+author: Scott Hammond
+date: 2015-05-08T18:00:00.000Z
+status: publish
+category: Community
+slug: transitions
+layout: blog-post.hbs
+---
+
+Em Fevereiro, nós anunciamos a [Fundação Node.js]
+(https://www.joyent.com/blog/introducing-the-nodejs-foundation),
+que irá administrar o futuro do Node.js e abri-lo para a
+comunidade de uma maneira que não estava disponível antes. Organizações como
+IBM, SAP, Apigee, F5, Fidelity, Microsoft, PayPal, Red Hat, e outras estão
+patrocinando a Fundação, e eles estão adicionando mais contribuidores ao projeto.
+A missão da Fundação é acelerar a adoção de Node e garantir
+que o projeto seja conduzido pela comunidade sob uma governança transparente e de modelo
+aberto.
+
+Sob a proteção da Fundação, o projeto Node.js está entrando para a próxima
+fase de maturidade e adotando um modelo em que não há nenhum BD ou líder de projeto.
```

Comment:
This sentence is weird... It does not make sense

Reply:
Can you please post the original one so we can find a better translation?

Reply:
Sure,

Under the aegis of the Foundation, the Node.js project is entering the next
phase of maturity and adopting a model in which there is no BD or project lead.

aegis means protection.
I think BD is Business Director, but i'm not sure about it, so I didn't change it.

Reply:
Did some changes, I think that's more like it. I really do not know how to translate BD to Portuguese too, so let's leave as it is